Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/MainAddActionsNavigationPage.xaml.cs

```diff
@@ -56,7 +75,16 @@ private void XNavPOM_Click(object sender, RoutedEventArgs e)
 
         private void XRecord_Click(object sender, RoutedEventArgs e)
         {
-            LoadActionFrame(new RecordNavPage(mContext), "Record", eImageType.Camera);
+            if (RecordPage == null)
+            {
+                RecordPage = new RecordNavPage(mContext);
+            }
+            else
+            {                
+                RecordPage.SetDefault(mContext);
```

Comment:
why to setDefualt each time the Record page is been selected by user? what if it already in middle of recording and user only went out of page and back?
we should do reset/detauls to page based on Context.PropertyChanged of Bf/CurrentActivity

Reply:
if the recording is on and if the user switches the page then the recording is stopped so the page is reset with context